.github/workflows/update.yml

```
@@ -6,7 +6,7 @@ name: Update
 
 on:
   schedule:
-    - cron: "0 11,23 * * *"
+    - cron: "0 6/6 * * *"
```

I'd find this equivalent expression easier to read.
```suggestion
    - cron: "0 6,12,18 * * *"
```